### ISSUE TITLE
Why are my mAP, Precision, and Recall charts have a high degree of variance?

### ISSUE THREAD

**Issue by erb13020** [ultralytics/yolov5#5104]
## ❔Question
Why are my metrics all over the place?
## Additional context
I'm training a new Yolo model to detect objects in civil litigation documents. I've trained Yolo models on this kind of data before, so the issue is most likely with something I'm doing wrong.

##Charts
**mAP_0.5:.95**
https://gyazo.com/1e826c49e1b939dfc6e35c651b0a86a8

**Recall, Precision, mAP_0.5**
https://gyazo.com/62f7226386862c83702e362785efc404

**Box Loss, Obj Loss, Cls Loss**
https://gyazo.com/ee494892fc62b41536381ed0be509982

**x/lr0, x/lr1, x/lr2**
https://gyazo.com/cbc45348c18300323b3515e3c588102b

##Command
`python3.8 train.py --data labelbox/labelbox.yaml --cfg models/hub/yolov5x6.yaml --weights '' --epochs 1600 --img 1280 --batch 12`

##Dataset

**Training Set**: 37 images
**Test Set**: 6 images
**Labels**: Roughly ~2500

I'm doing a proof of concept before training on a much larger dataset appropriate for Yolov5.

**Comment by glenn-jocher** [comment]
@erb13020 low statistics?

You might want to start from a pretrained model in any case.

**Comment by erb13020** [comment]
@glenn-jocher 

I ended up using the pretrained model and it's training normally and mAP_0.5:.95 increased from 0.72 to 0.8
